binding/android/OrcaTestApp/orca-test-app/build.gradle

```diff
@@ -106,7 +106,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.6.1'
     implementation 'com.google.android.material:material:1.8.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
-    implementation 'ai.picovoice:orca-android:0.1.0'
+    implementation files('../../Orca/orca/build/outputs/aar/orca-release.aar')
```

Comment:
at this point, I'd recommend staging the SDK on Maven Central so that you can see how it works in the real world. Not sure if you've done it before, but it is explained in the runbook, or you can ask Kwangsoo or Eric